cmd/priv_val_server/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	cmn "github.com/tendermint/tmlibs/common"
+	"github.com/tendermint/tmlibs/log"
+
+	priv_val "github.com/tendermint/tendermint/types/priv_validator"
+)
+
+func main() {
+	var (
+		chainID     = flag.String("chain-id", "mychain", "chain id")
+		numClients  = flag.Int("clients", 1, "number of concurrently connected clients")
+		privValPath = flag.String("priv", "", "priv val file path")
+		socketAddr  = flag.String("socket.addr", ":46659", "socket bind addr")
+
+		logger = log.NewTMLogger(log.NewSyncWriter(os.Stdout)).With("module", "priv_val")
+	)
+	flag.Parse()
+
+	logger.Info("Reading args privValidatorSocketServer", "chainID", *chainID, "privPath", *privValPath)
```

Comment:
Maybe change to `fmt.Sprintf("Starting private validator socket server on %s", listenAddr), "chainID", *chainID, "privValPath", *privValPath,` ?